content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
 {{% /tutorials/step %}}
 {{% tutorials/step title="Add a metrics data source" %}}
 
-The sample application exposes metrics which are stored in [Prometheus](https://prometheus.io/), a popular time series database.
+The sample application exposes metrics which are stored in [Prometheus](https://prometheus.io/), a popular time series database (TSDB).
 
 To be able to visualize the metrics from Prometheus, you first need to add it as a data source in Grafana.
 
-- In the side bar, click **Configuration** -> **Data Sources**.
-- Click **Add data source**, and select "Prometheus" from the list of available data sources.
-- In the URL box, type "http://prometheus:9090".
-- Click **Save & Test** to save your changes.
+- In the side bar, hover your cursor over the  **Configuration** (gear) icon, and then click **Data Sources**.
+- Click **Add data source**. 
+- In the list of data sources, click **Prometheus**.
+- In the URL box, type **http://prometheus:9090**.
+- Click **Save & Test**.
 
 Prometheus is now available as a data source in Grafana.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Explore your metrics" %}}
 
-Grafana Explore, introduced in Grafana 6.0, is a workflow for troubleshooting and data exploration. In this step, you'll be using Explore to understand the metrics exposed by the sample application.
+Grafana Explore is a workflow for troubleshooting and data exploration. In this step, you'll be using Explore to understand the metrics exposed by the sample application.
 
-- In the side bar, click **Explore**.
-- In the **Query** box, type the following, and press Enter:
+- In the side bar, click the **Explore** (compass rose) icon.
+- In the **Query editor**, where it says *Enter a PromQL query*, enter `the following text` and then press Enter.
```

Comment:
```suggestion
- In the **Query editor**, where it says *Enter a PromQL query*, enter the following text and then press Enter.
```

"the following text" refers to:

```
tns_request_duration_seconds_count
```

Reply:
ARGH. Guess I was more distracted yesterday than I thought. Sorry.